### DISCUSSION TITLE
Ocean Modeling Collaboration Bi-monthly Meeting - legacy

### DISCUSSION THREAD

**Discussion by saeed-moghimi-noaa** [oceanmodeling/Management#15] [Announcements]
### **Time zone:**  America/New_York
Google Meet joining info
Video call link: https://meet.google.com/eyy-zqbd-ppr
Or dial: ‪(US) +1 208-715-5331‬ PIN: ‪260 201 088‬#
More phone numbers: https://tel.meet/eyy-zqbd-ppr?pin=9418066551126 

### Agenda:
- Review Ocean Modeling current projects:
  * *  Searvey status
-  Talk about potential new projects
  * * Unstructured model output subsetting 
  * *  Pangeo based visualization package (e.g. Thalassa)

I hope to see you.
@saeed-moghimi-noaa  and @brey

**Comment by saeed-moghimi-noaa** [comment]
## Jun 1, 2023 
### [Ocean Modeling Collaboration Bi-monthly Meeting](https://www.google.com/calendar/event?eid=am85ODYwZjA1c2UzbzUxZHZwbnRjZnZqOXNfMjAyMzAyMDJUMTQwMDAwWiBzYWVlZC5tb2doaW1pQG5vYWEuZ292)

### Notes:
Review Ocean Modeling current projects:
* Searvey status
  * Searvey released 
  * Binder to test it
  * USGS 
      * API is not always consistent
      *  Number of stations changes for the same ipython sessions with the same request 
  *  IOC 
      * Api needs to be checked to make sure their changes not breaking our work

* UFS-Coastal and CoastalApp
  * FVCOM is merged into CoastalApp
  * FVCOM test cases in added to CoastalAppTestSuites
  * SCHISM is included into ufs-coastal
  * ADCIRC cap has issues regarding global vs local nodes
  * Start testing SCHISM from ufs-coastal
     * To get more heavy duty test cases from SCHISM developers
     * NCAR will reach out to developers for more questions
  * Hera, Orion and Frontera are the main
  * For compiling codes, we need keep an eye on Spack for managing dependencies


* Question on merging the two repos schism-esmf + schism
   * Carsten has several reservations (kept here fore reference, let's discuss this later)
   * different developer pool
   * different build system
   * different continuous integration
   * different documentation structure
   * easy to deal with two repos, or subrepoing/submoduling
   * much harder to disentangle after a merge.  High granularity is good.
 
* Talk about potential new projects
   * Unstructured model output subsetting 
      * To have it a s service and solve it in client side
  * Pangeo based visualization package (e.g. Thalassa)
    * Split Thalassa in two parts
  * Mesh generation (river + coastal)
  * Relocatable 3D modeling

**Comment by tomsail** [comment]
## Reminder about Ocean Modeling Collaboration Bi-monthly Meeting

Dear oceanmodeling community: 
@oceanmodeling/meshgen @oceanmodeling/model-wrapper @oceanmodeling/observational-data-retrieval @oceanmodeling/secofs @oceanmodeling/ufs-coastal 

The next bi-monthly meeting is planned this Thursday. Here is the **draft** agenda:
* data retrieval 
  * Searvey update: NDBC by @SorooshMani-NOAA 
* Model preparation and meshing: 
  * new member @veenstrajelmer (or @Tammo-Zijlker-Deltares) will present [dfm_tools](https://github.com/Deltares/dfm_tools), a Python package for pre- and postprocessing D-FlowFM model input and output files
  * @krober10nd will present [his work](https://github.com/krober10nd/SMSOceanMeshToolkit/tree/tool-development) on the oceanmesh plugin for SMS community version:
* Model visualisation: 
  * @tomsail will present an interactive dashboard workflow for viewing the results of the global surge model in development at JRC. [code](https://github.com/seareport/skill-panel-demo) and [demo](https://skill-panel-3607dfd2cee1.herokuapp.com/) already available

--- 
Details for connecting to the meeting: 

When: Thursday, October 3rd · 09:00 – 10:30 AM
Time zone: America/New_York
Google Meet joining info
Video call link: Meeting link
[meet.google.com/eyy-zqbd-ppr](https://meet.google.com/eyy-zqbd-ppr)
Join by phone
(US) [+1 208-715-5331](tel:+1-208-715-5331%3B260201088%23)
PIN: 260201088
More phone numbers: https://tel.meet/eyy-zqbd-ppr?pin=9418066551126

@saeed-moghimi-noaa and @tomsail 

--- 
Feel free to reply and ping external people likely to be interested by this call (@sebourban, @nicogodet)

**Reply by tomsail** [reply]
precision on the time zone for the meeting: 
https://www.timeanddate.com/worldclock/fixedtime.html?msg=oceanmodeling+bimonthly+meeting+&iso=20241003T09&p1=179&ah=1&am=30

**Reply by tomsail** [reply]
All, please find the presentations showed at the meeting: 
@veenstrajelmer: https://github.com/Deltares/dfm_tools/blob/main/docs/dfm_tools.pdf
@tomsail: https://github.com/seareport/skill-panel-demo/wiki

**Reply by tomsail** [reply]
@veenstrajelmer if you can please put again the link of the notebooks you shared in the jitsi; I lost all message when leaving the conversation.

**Reply by veenstrajelmer** [reply]
Sure!

I mainly wanted to share the [modelbuilder example notebook](https://github.com/Deltares/dfm_tools/blob/main/docs/notebooks/modelbuilder_example.ipynb) since it shows the python code behind what I presented. I realize we use quite some tools that are comparable to initiatives being developed by other institutes. I guess that is what this group is for, to raise awareness about initiatives and to align where possible.

Recently I also worked on [subset/retrieval of observation waterleveldata](https://github.com/Deltares/dfm_tools/blob/main/docs/notebooks/subset_retrieve_sealevel_observations.ipynb), with several sources retrieveable via the same type of code resulting in netcdf files that are formatted the same. I will definately look into searvey to look for overlap. This is just a proof of concept, I'd like to move it to a separate (existing) package eventually.

**Comment by tomsail** [comment]
# Bi-monthly Meeting December 2024

Dear @oceanmodeling/everyone, 

The next bi-monthly meeting is next week, on Thursday 5th of December.

We'd already have some items to push on the agenda: 
 * new packages on oceanmodeling ( seastats ,  inspectds and xarray-selafin)
 * new repo: a demonstration for a reproducible dashboard for skill-assessment (simplified version that everyone can try out).
 * [To be confirmed] Presentation of @acerrone3 and his work. (Not sure if the ping works, just in case @SorooshMani-NOAA can you please forward it to him?).
 * @saeed-moghimi-noaa and others let me know if there is any more items to push forward! 
 
 I'll set up a Jitsi meeting to avoid any admin problems, like last time (info about jitsi: https://jitsi.org/)

**Reply by tomsail** [reply]
Dear all, here is the updated and confirmed agenda for tomorrow at 15:00 CET: 

[check this [link](https://www.timeanddate.com/worldclock/fixedtime.html?msg=oceanmodeling+bimonthly+meeting&iso=20241205T15&p1=48&ah=1&am=30) for your local time zone]

 * new packages on oceanmodeling ([seastats](https://github.com/oceanmodeling/seastats), [xarray-selafin](https://github.com/oceanmodeling/xarray-selafin) and [inspectds](https://github.com/oceanmodeling/inspectds))
 * presentation by @pmav99 of a reproducible dashboard (with sample data) for assessing model vs observations skill.
 * presentation by @acerrone3 of his work on STOFS2D Development model. 
 * discussions
 * discussion on a new agenda for meeting (for more accessibility and control for our community)

I will send here (and by mail) a jitsi link 1 hour prior to the meeting. 

See you all tomorrow

**Reply by tomsail** [reply]
Dear all, 
Here are the details for connecting to the meeting:

When: Thursday, October 3rd · 15:00 – 16:30 CET // 09:00 – 10:30 EST
[Check your local time zone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=oceanmodeling+bimonthly+meeting&iso=20241205T15&p1=48&ah=1&am=30) 

Video call link: https://moderated.jitsi.net/9c6c9b161cc645afaeca69ff075dad507bc1b26e328b4bad952bc55574ef0fe7

**Reply by tomsail** [reply]
Please find the presentation for the updates on new packages [oceanmodeling_20241205_updates_TSR.pdf](https://github.com/user-attachments/files/18025511/oceanmodeling_20241205_updates_TSR.pdf)

**Reply by tomsail** [reply]
And as @acerrone3 asked, here is the work of @vvoukouvalas on the data-driven model for daily max Water Levels.
https://www.frontiersin.org/journals/marine-science/articles/10.3389/fmars.2024.1429155/full

**Comment by platipodium** [comment]
Is there a calendar file we can subscribe to?

**Reply by brey** [reply]
@platipodium There is the a google calendar invitation that @saeed-moghimi-noaa has set up since the beginning.

I see that you were invited but maybe you are not synced.

If anyone has a better idea on how to setup a calendar so that we don't impose on a specific person, let us know.

**Reply by platipodium** [reply]
Hmm, so if I lost that initial invite how can I recover it?

**Reply by brey** [reply]
@saeed-moghimi-noaa, can you please invite again all so that people can catch up with the meetings. We have to update the list of people invited with the calendar invitation with the list of oceanmodelling/everyone (see above in the message from @tomsail).

**Comment by hga007** [comment]
I always have conflicts with these bi-monthly meetings. Thursdays are not suitable for me because of my commitments with JEDI. Also, I still need to install another free conference software (Jitsy) on my computer or iPad. I already have four different conference applications. Why there is a need for a new one?

**Reply by tomsail** [reply]
Hi Hernan thanks for your feedback.
 * For Jitsi you should be able to join directly in the browser (I just tried successfully with my iPhone).
 * We are in the process of switching to a more "community oriented" agenda and calendar. @brey will post shortly instructions about this transition. 
 * Proposing another date & time for meetings, in order to accommodate more people (and time zones, hello @simonweppe) around the world is definitely foreseen. But change will be driven by active members when the discussion arise.
 
 Looking forward seeing you in the next meetings

**Reply by saeed-moghimi-noaa** [reply]
Hi Hernan, Thanks for the comment! Understood! -Saeed

**Reply by simonweppe** [reply]
Thanks @tomsail ! Please sign me in for the metings (mailing list or invite?), would love to stay in touch with your dev, and contribute when we can we use cases.

**Comment by hga007** [comment]
Excellent, thank you. We require authorization from the cyberinfrastructure managers to install any free software on the university computers or browsers. We can only use their authorized Zoom software when representing our institution. We are constantly under attack from unwanted entities. I assume that this is the policy everywhere.

**Comment by hga007** [comment]
By the way, we require several signatures from cyberinfrastructure to distribute ROMS over GitHub.  We were negated twice and won on the third try.

**Comment by brey** [comment]
@oceanmodeling/everyone Please find below the new arrangement for the meeting management:


We have now a functional email box : oceanmodelingcollaboration@gmail.com


We also have a corresponding calendar with a public url : https://calendar.google.com/calendar/u/0?cid=b2NlYW5tb2RlbGluZ2NvbGxhYm9yYXRpb25AZ21haWwuY29t

To be able to post events yourself and integrate with your calendar, please send an `email` to the above functional email specifying your needs:

- Add me to the bi-monthly meeting

and/or

- I want to post calendar events
